configure.ac

```diff
@@ -653,9 +654,9 @@ AS_CASE([$host],
     [exeext=".exe"],
   [exeext=''])
 
-otherlibraries="dynlink runtime_events"
+dnl otherlibraries="dynlink runtime_events"
+otherlibraries="dynlink"
 lib_dynlink=true
-lib_runtime_events=true
```

Comment:
Sorry, I should have posted that earlier:
I wonder whether it would be safer to
simply `dnl` this line?

Reply:
Sure, I've left CRs at this point too now.

Reply:
Indeed, I should have suggested
that, without the CRs, the `dnl`
provides little safety.